tests/unit/tests.sh

```diff
@@ -71,7 +71,7 @@ T_procfile-types() {
     expected="Procfile declares types -> web, worker"
     actual="$(procfile-types invalid-proc | tail -1)"
 
-    if [[ $actual != $expected ]]; then
+    if [[ $actual != "$expected" ]]; then
```

Comment:
Do we need quotes around `$actual` as well?

Reply:
Not according to `shellcheck`

https://github.com/koalaman/shellcheck/wiki/SC2053

Reply:
I would just add them there, seems silly to not.

Reply:
done